src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+
+<!--- TODO image of properties window --->
+
+## Changing the Play Zone
+
+The "Play Zone" is a cylindrical area where the player will *not* receive constant damage. The Play Zone will shrink in size for a certain amount of intervals.
+
+### Change the Play Zone Settings
+
+Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone** and open the **Properties** window. The object has many **Custom Properties** that will be described below:
+
+| Property Name | Description |
+| --- | --- |
+| Phase Count | The number of phases. The Play Area will shrink in size as the phases increase. It will stop shrinking once the last phase occurs. |
+| ZoneSizeRatio | The ratio for the Play Area to shrink after each phase. For example, `0.7` signifies the area will shrink down to 70% of its current phase size. |
+| StaticTime | The amount of seconds each phase will last before it starts shrinking. |
+| Closing Time | The amount of seconds it takes to shrink before starting the next phase. |
+| BaseDamageRate | The amount of damage per second applied to the players outside the Play Zone during phase 1. |
+| DamageMultiplier | The amount multiplied to BaseDamageRate after each phase. |
+| UseRoundTiming | The toggle to restart the Play Zone after each round. |
+| ActivationDelay | The amount of seconds until the first phase begins. |
+
+### Change the Initial Play Zone Size
+
+1. Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone > ServerContext**.
+2. Select the **InitialZone** object.
+3. Focus on the cylinder mesh by pressing ++F++.
+4. Press ++R++ to enter scale mode.
+5. Change the X and Y scale so the initial Play Zone is a different size.
+
+<!--- TODO image of smaller play zone --->
+
+## Changing the Loot Tables
+
+There are several LootSpawner object placed throughout the map that randomly roll a weapon from the Loot Table list. Each roll on the Loot Table has an equal chance to be rolled.
+
+### Add a New Roll to the Loot Table
+
+Navigate to **Gameplay Settings > Basic Loot Table > ServerContext > Loot** group and notice there are three objects inside representing the three different weapons in the Loot Table. Select one of the **Object (server)** and duplicate it using the keyboard shortcut ++Ctrl++ + ++W++.
+
+<!--- TODO image of hierarchy for new object --->
+
+### Change Weapon in the Loot Table
+
+1. Select the new **Object (server)**.
+2. Open the **Properties** window and find the **LootTemplate** property.
+3. Now open the **Core Content** and search for `Basic Submachine Gun` template.
+4. Drag and drop the submachine gun template into the **LootTemplate** property.
+
+<!--- TODO image of properties for new weapon --->
+
+## Learn More
+
+[Publishing Your Game](publishing.md) | [Dungeon Crawler Framework Reference](dungeon_framework.md) | [Environment Art Tutorial](environment_art.md) | [Weapons Tutorial](weapons.md)
```

Comment:
THat weapons doc is a reference. Generally, if the file is just named after the feature, it's the reference. Environment Art is an exception, but we'll fix that someday. Would swap the Dungeon to the tutorial, not the reference, for more of that flow chart what-tutorial-should-I-do next.

Also order for these (in a tutorial) should be:

References you really need to finish this > References that are linked throughout it > Something to do next

So almost exactly how you have it, but Dungeon last.